packages/playwright-core/src/protocol/channels.ts

```diff
@@ -3788,7 +3788,15 @@ export type AndroidDeviceInputDragOptions = {
 };
 export type AndroidDeviceInputDragResult = void;
 export type AndroidDeviceLaunchBrowserParams = {
-  pkg?: string,
+  noDefaultViewport?: boolean,
```

Comment:
Mixed feelings about it: we'd rather see users emulate android using linux than emulate android device using android.